dist/formrenderer.standalone.uncompressed.js

```diff
@@ -871,7 +871,7 @@ rivets.configure({
       return this.get('min_max_length_units') || 'characters';
     },
     setExistingValue: function(x) {
-      if (x) {
+      if (x != null) {
```

Comment:
Do we have to check for null, because if we check for falsey it won't pick up `0` as a valid field value?

Reply:
We want to only set the value if a value exist. `false` is an acceptable value and this is the reason why it was sometimes not persisted as part of the response hash.